tests/baselines/reference/enumWithExport.types

```diff
@@ -10,6 +10,6 @@ enum x {
 >x : x
 
   z = y
->z : x.z

```

Comment:
interesting change here: `y` is not resolved so `z` gets the type of the parent enum instead of an enum member type. I think that's fine, but I wouldn't have predicted it from reading the code.

Reply:
Actually, it's given the unique enum member type `x.z` associated with a computed member. But we display the type as just `x` because there is only one member (i.e. type `x`, the union of all enum members, and type `x.z` are the same).